enable_autologin.yml

```diff
@@ -18,3 +19,13 @@
         owner: root
         group: root
         mode: '0644'
+    - name: check if p is currently running
+      ansible.builtin.command: pgrep --uid p
```

Comment:
I think you can use https://docs.ansible.com/ansible/latest/collections/community/general/pids_module.html and for the next step check if the list is empty to use a more ansible-builtin method

Reply:
does not appear that pids module allows searching by uid.